### ISSUE TITLE
The appearances name in the .Material.json files does not respect the .mesh content

### ISSUE THREAD

**Issue by Neurolinked** [WolvenKit/WolvenKit#1916]
**Describe the bug**
In the .Material.json file exported from a .mesh has in the appearance list the appearance names wrong respectively from the content of the .mesh file. some digits are added to the end.

**To Reproduce**
1. Go to the Asset Browser
2. search for base\vehicles\sport\v_sport1_rayfield_caliburn\entities\meshes\v_sport1_rayfield_caliburn__ext01_body_01.mesh
3. Open it, even without importing it in the mod
4. Verify the names of the Appearances of the model
5. Export the mesh with the materials
6. Look into the exported .Material.json file related to the model at the Appearances section
7. You will see the appearances names with a number at the end. That number feels like an index and it's not supposed to be there

**Expected behavior**
In the .Material.json files you'll find the same names as the appearances are in the .mesh file

**Version (please complete the following information):**
Wolvenkit 8.14.0 and Wolvenkit.CLI 8.14.1

**Screenshots**
![mesh_file](https://github.com/user-attachments/assets/c34410ea-c51d-4a24-8193-c956cb2b357a)
![material_json](https://github.com/user-attachments/assets/c43578bb-f4ba-48a9-8432-6c7f7affed0a)

[v_sport1_rayfield_caliburn__ext01_body_01.Material.json](https://github.com/user-attachments/files/16847957/v_sport1_rayfield_caliburn__ext01_body_01.Material.json)

**Comment by seberoth** [comment]
Yeah, its an index. Jack added it while working on the preview. Don't know if it's still needed.
Will check.

**Comment by Simarilius-uk** [comment]
we use that info in the blender plugin

**Comment by kmirea** [comment]
The incremental suffix is afaik intended to address the issue of materials that share the same name, even same base, but have different parameters, sometimes even within the same entity / mesh collection - such as vehicles. it wreaked havoc, particularly in blender, and material depot refs.  I reckon this could be addressed perhaps by attaching the original material naming within the glbs extradata, and restoring original names either on export from blender, or reimport to wkit
